content/docs/2.10/operate/validating-webhook.md

```diff
@@ -0,0 +1,16 @@
++++
+title = "Validating Webhook"
+description = "Details on KEDA Validating Webhook"
+weight = 100
++++
+
+## Validating Webhooks
+
+Webhooks require a valid certificates to expose the server for the api-server. To achieve this KEDA generates it's own certificates and stores them in a secret. This solves the problem but some users could want to have their own certificates generated from their own CA in order to improve the security. This can be doable disabling the certificate rotation via setting the console argument `disable-cert-rotation=true` and placing the `tls.crt` and `tls.key` files (these names are required by the framework and can't be changed) in the path `/certs`. The path is also configurable using the console argument `webhooks-cert-dir=PATH-FOR-CERTS`.
```

Comment:
```suggestion
The KEDA Admission Controller requires a valid certificate to expose the server for the api-server. To achieve this KEDA generates its own certificates and stores them in a secret. 

While this is a good starting point, some end-users may want to use their own certificates which are generated from their own CA in order to improve security. This can be achieved by disabling the certificate rotation via setting the console argument `disable-cert-rotation=true` and mounting the `tls.crt` and `tls.key` files to `/certs`. The path is also configurable using the console argument `webhooks-cert-dir=PATH-FOR-CERTS`.
```